jupyter_development/utils.py

```diff
@@ -0,0 +1,91 @@
+import os
```

Comment:
Is this script plus all the notebooks to be merged?  Aren't they just dev stuff?

Reply:
they are, I thought about making them gists but then went with having everything in one place for collaboration, they should go and be turned into examples I think